src/our-work.html

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: Our Work
+---
+
+<section class="row position-relative">
+  <div class="container">
+    <div class="offset-lg-1 col-lg-7 py-5 my-5 py-md-0 my-md-0">
+      <span class="pill">Our Work</span>
+      <h1 class="mt-4 mb-3">Building solutions that empower government services</h1>
+      <p class="mb-4 text-white">
+        We are committed to building open-source, accessible solutions to elevate government workers through teaching and training
+        to ensure their success.
+      </p>
+      <a class="btn btn-primary me-lg-3 mb-3 mb-lg-0" href="mailto:hello@compiler.la">Contact us</a>
+      <a class="btn btn-outline-primary" target="_blank" href="/capabilities">Capabilities statement</a>
```

Comment:
Capabilities statement is sentence-case here but title-cased in the menu.

Reply:
Yep, here we followed the mockup in Figma and it looks like button titles are sentence-case but we can review it in case we need to make the menu and the buttons consistent.